protocol/protobuf/pairing.proto

```diff
@@ -136,10 +136,11 @@ message SyncInstallationCommunity {
   SyncCommunitySettings settings = 9;
   bool encrypted = 10;
   bool spectated = 11;
-  bytes encryption_keys = 12;
+  bytes deprecated_encryption_keys = 12;
```

Comment:
why not mark as deprecated? 🤔 
```suggestion
  bytes encryption_keys = 12 [deprecated = true];
```

and name the new one like `encryption_keys_v2` or smth

Reply:
Because... what are we gonna do if we decide to deprecate `encryption_keys = 16`. 
Rename `deprecated_encryption_keys` to `deprecated_deprecated_encryption_keys`? 😄